garnett/context.py

```diff
@@ -1,23 +1,17 @@
 from contextlib import ContextDecorator
 import contextvars
 
-from garnett.utils import get_default_language
-
-ctx_language = contextvars.ContextVar("garnett_language")
+# Internal context var should be set via set_field_language and get via get_current_language
+_ctx_language = contextvars.ContextVar("garnett_language")
 
 
 class set_field_language(ContextDecorator):
-    def __init__(self, language, deactivate=False):
-        self.old_language = ctx_language.set(language).old_value
-        if self.old_language == contextvars.Token.MISSING:
```

Comment:
What happens if there was no previous language?

Reply:
This is handled by the `get_current_language` function which just returns the default language